packages/core/test/test-util/test-services/v4/test-service/function-imports.ts

```diff
@@ -147,54 +127,6 @@ export function testFunctionImportComplexReturnTypeCollection(parameters: TestFu
   return new FunctionImportRequestBuilder('get', '/sap/opu/odata/sap/API_TEST_SRV', 'TestFunctionImportComplexReturnTypeCollection', (data) => transformReturnValueForComplexTypeList(data, TestComplexType.build), params);
 }
 
-/**
```

Comment:
Why were these test cases removed? Is this the same in v2 and v4? Perhaps it also makes sense to have a edmx common and then a v2 v4 edmx for testing specific things?

Reply:
Yeah, maybe you're right - it is not 100% the same and for regressions it is good to have it.

Reply:
Ah, also FunctionImports in v4 can only be GET

Reply:
First, I was actually thinking that one of the functions could make sense (createTestComplexType because its name conflicts with the testcomplextype builder), but I just checked and this is done in the common part, so I don't see the necessity to have an additional test here.
Secondly, Functions in v4 can only be GET therefore I removed the other test cases.